samples/javascript_nodejs/25.logger-bot/README.md

```diff
@@ -72,3 +74,5 @@ msbot clone services -f deploymentScripts/msbotClone -n myChatBot -l <Azure-loca
 [8]: https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-v4-storage?view=azure-bot-service-4.0&tabs=jsechoproperty%2Ccsetagoverwrite%2Ccsetag
 [9]: https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-v4-state?view=azure-bot-service-4.0&tabs=js
 [10]: https://dev.botframework.com
+[11]: https://github.com/southworkscom/BotBuilder-Samples/blob/master/Installing_CLI_tools.md#install-cli-tools
```

Comment:
As you can see [here](https://github.com/Microsoft/BotBuilder-Samples/blame/master/samples/javascript_nodejs/06.using-cards/README.md#L47) we tend to use the relative path to the `Install CLI Tools` file, taking advantage it's in the root folder of the repo, and the user has to clone the entire repo to try the sample.
```suggestion
[11]: ../../../Installing_CLI_tools.md
```